api/api_keys/serializers.py

```diff
@@ -11,17 +14,21 @@ class MasterAPIKeySerializer(serializers.ModelSerializer):
         "for every endpoint apart from create",
     )
     is_admin = serializers.BooleanField(default=True)
+    if settings.IS_RBAC_INSTALLED:
+        roles = MasterAPIKeyRoleSerializer(many=True, read_only=True, source="role")
```

Comment:
Hmm... I don't really like this behaviour of changing serializers based on the existence of external libraries. Can we think of another way to do this? Perhaps, as per the suggestion on the other PR, by creating the view in the rbac repo and adding it to URLs in the core repo. We do this for a number of views in the flagsmith-workflows repo for example.

Reply:
I saw that you had a similar comment in another PR, yes I think you are right, the serializers would no longer depend on RBAC and could be tested more easily

Reply:
So can we remove it from here now?